lib/livebook/application.ex

```diff
@@ -61,7 +65,11 @@ defmodule Livebook.Application do
         clear_env_vars()
         display_startup_info()
         Livebook.Hubs.connect_hubs()
-        deploy_apps()
+
+        unless serverless?() do
```

Comment:
```suggestion
        if Process.whereis(LivebookWeb.Endpoint) do
```

Reply:
Maybe it is a nitpick but, instead of scattering the same conditional check, we check for `serveless?` once and then assert on its "results".

Reply:
Hmm, this feels like an additional level of indirection to me.

Reply:
That's also fair, your call. :)